CHANGELOG.md

```diff
@@ -6,8 +6,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- Support for `.preset_memo_wise` on class methods
+- Support for `.reset_memo_wise` on class methods
 ### Updated
-  Improved perfomance for common cases by reducing array allocations
+- Improved performance for common cases by reducing array allocations
+- Shared test setup and cases across instances / class `self.` / `class << self`
```

Comment:
I'm not sure this is useful for outside readers but I feel like we may have had this conversation about other changelog items in the past and just disagreed so I'm happy to keep this here if you like it.

Reply:
I can remove it if you prefer!

Reply:
Removed!